pallene/parser.lua

```diff
@@ -249,7 +249,8 @@ local grammar = re.compile([[
     toplevelrecord  <- (P  RECORD NAME^NameRecord recordfields
                            END^EndRecord)                        -> ToplevelRecord
 
-    localopt        <- (LOCAL)?                                  -> opt_bool
+    modifier        <- LOCAL -> to_true
+                     / EXPORT -> to_false
```

Comment:
What do you think about using a more specific name such as `export_modifier` instead of `modifier`?

I also wonder if the rest of the code would become clearer if we changed `is_local` to `is_exported`.

Reply:
>What do you think about using a more specific name such as export_modifier instead of modifier?
The modifier rule has both the `local` and `export` keywords. It is somewhat confusing to have them both under a rule called `export_modifier`.

> I also wonder if the rest of the code would become clearer if we changed `is_local` to `is_exported`.
As far as I can tell, it is clear. I think we could use the time and energy on something else. :P

Reply:
What about `export_or_local`? What I don't like about the "modifier" name is that it is too vague.